pkg/db/db.go

```diff
@@ -50,11 +51,21 @@ func FromEnv(path string, backend string) (datab, error) {
 	if path == "" {
 		return nil, fmt.Errorf("missing DB_PATH")
 	}
-
-	c, err := newDB(Config{
-		Type: backend,
-		Path: path,
-	})
+	var (
+		c   datab
+		err error
+	)
+	if useCloudSQL {
+		c, err = NewCloudSQL(Config{
+			Type: backend,
+			Path: path,
+		})
+	} else {
+		c, err = newDB(Config{
+			Type: backend,
+			Path: path,
+		})
```

Comment:
Since the `Config` is the same for both you can init it before the `if/else` and then just pass it in